content/swift/basic-operators-in-swift.md

```diff
@@ -0,0 +1,198 @@
+<!--
+---
+Title: Basic Operators in Swift
+Subjects: # Please first look to use Subjects that are already listed at author.codecademy.com/browse/categories
```

Comment:
Please make the subjects and tags all lowercase. We just decided on this formatting today, so no worries at all for having them uppercase here.